StratisCore.UI/src/app/setup/create/confirm-mnemonic/confirm-mnemonic.component.ts

```diff
@@ -18,16 +19,18 @@ export class ConfirmMnemonicComponent implements OnInit {
 
   public secretWordIndexGenerator = new SecretWordIndexGenerator();
 
-  constructor(private apiService: ApiService, private genericModalService: ModalService, private route: ActivatedRoute, private router: Router, private fb: FormBuilder) {
+  constructor(private apiService: ApiService, private genericModalService: ModalService, private route: ActivatedRoute, private router: Router, private fb: FormBuilder, private globalService: GlobalService) {
     this.buildMnemonicForm();
   }
   private newWallet: WalletCreation;
   private subscription: Subscription;
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use `Boolean` (the one with capital B), because 
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
No idea why I used Boolean here, I use boolean everywhere else. Thanks!